AmazonChimeSDK/AmazonChimeSDK/internal/analytics/DefaultMeetingStatsCollector.swift

```diff
@@ -11,6 +11,7 @@ import Foundation
 @objcMembers public class DefaultMeetingStatsCollector: NSObject, MeetingStatsCollector {
     private let logger: Logger
     private var meetingStartTimeMs: Int64 = 0
```

Comment:
meetingStartTimeMs is the time that meeting is started,
meetingStartConnectingTimeMs is the time that meeting start request is initialized, right?

Reply:
yes